codelists/models.py

```diff
@@ -26,7 +26,7 @@ class Codelist(models.Model):
     methodology = models.TextField()
 
     class Meta:
-        unique_together = ("project", "slug")
+        unique_together = ("project", "name")
```

Comment:
We need to keep `unique_together = ("project", "slug")`.  If two codelists are created in the same project called "George's list" (your list) and "Georges list" (a list of people called George -- hey, it could happen) then they'll both get the same slug, and so the `get_object_or_404` (which uses the slug) will find more than one object.

Reply:
Good point!  Fixed in db3cdfd

> a list of people called George

There's at least two of us!

Reply:
I've hit upon the same problem as `project` with `slug` - it's not included in the form so we have to add it so we can validate uniqueness.  But that brings up a UX issue - say `George's list` already exists and someone else wants to add `Georges list`.  They'll get an error on the page which says something like "Codelist must have a unique Project, Name, and Slug."  However, they can't change the slug we generate and are left with modifying the Title.

Is that the flow we want to push for?  If not should we look at updating the slug to avoid a known collision?

Reply:
This is rare enough that I think it's acceptable to tell the user that there's a duplicate slug, and to leave them to work out the rest.

Reply:
Fixed in 62bc490